doc/source/index.rst

```diff
@@ -46,11 +46,17 @@ to implement custom failure criteria and computation.
 
         Provides developer installation and usage information.
 
+The minimum set of inputs to run a postprocessing for composites is the result files of an
+MAPDL solution and a material file (MATML) which was generated by Engineering Data.
+Lay-up files from ACP are optional and only required for some advanced operations. Refer to
+:ref:`Limitations`.
+
```

Comment:
Have you tried that a MatML file from engineering data is enought. I'm not sure if the material uuids are then written.